src/app.component.tsx

```diff
@@ -33,13 +35,38 @@ const assets: Assets = {
 
 export default class App extends React.Component {
 
+  private getActiveRouteName = (navigationState): string | null => {
```

Comment:
we have this code somewhere in navigation utils

Reply:
Done.